frontend/src/lib/components/Subscriptions/views/EditSubscription.tsx

```diff
@@ -280,140 +273,152 @@ export function EditSubscription({
+                                                {({ value, onChange }) => (
+                                                    <LemonSelect
+                                                        {...commonSelectProps}
+                                                        dropdownMatchSelectWidth={false}
+                                                        options={monthlyWeekdayOptions}
+                                                        // "day" is a special case where it is a list of all available days
+                                                        value={value ? (value.length === 1 ? value[0] : 'day') : null}
+                                                        onChange={(val) =>
+                                                            onChange(
+                                                                val === 'day' ? Object.keys(weekdayOptions) : [val]
+                                                            )
+                                                        }
+                                                    />
+                                                )}
+                                            </Field>
+                                        </>
                                     )}
-                                </Field>
+                                    <span>by</span>
+                                    <Field name={'start_date'}>
+                                        {({ value, onChange }) => (
+                                            <LemonSelect
+                                                {...commonSelectProps}
+                                                options={timeOptions}
+                                                value={dayjs(value).hour().toString()}
+                                                onChange={(val) => {
+                                                    onChange(
+                                                        dayjs()
+                                                            .hour(typeof val === 'string' ? parseInt(val, 10) : 0)
+                                                            .minute(0)
+                                                            .second(0)
+                                                            .toISOString()
+                                                    )
+                                                }}
+                                            />
+                                        )}
+                                    </Field>
+                                </div>
                             </div>
-                        </div>
-                    </section>
-                )}
+                        </section>
+                    )}
+                </div>
 
-                <footer className="flex justify-between pt-4">
+                <footer className="flex justify-between pt-4 border-t">
```

Comment:
Yeah this will actually be superseded by the new modals. The current modals have weird implicit styling on <section> which (as I changed some of the layout) removed the border hence `border-t` which is `border-top-width: 1px`